ecs-cli/modules/cli/compose/entity/service/service_test.go

```diff
@@ -418,6 +450,80 @@ func TestCreateWithHealthCheckGracePeriodAndELB(t *testing.T) {
 		func(healthCheckGracePeriod *int64) {
 			assert.Equal(t, int64(healthCheckGP), *healthCheckGracePeriod, "HealthCheckGracePeriod should match")
 		},
+		func(strategy []*ecs.PlacementStrategy) {
+			assert.Equal(t, 1, len(strategy), "PlacementStrategy should have size 1")
+			assert.Equal(t, ecs.PlacementStrategyTypeRandom, *strategy[0].Type, "PlacementStrategy should be random")
+		},
+	)
+}
+
+func TestCreateWithRandomPlacementStrategy(t *testing.T) {
+	placementStrategyFlag := ecs.PlacementStrategyTypeRandom
+	flagSet := flag.NewFlagSet("ecs-cli-up", 0)
+	flagSet.String(flags.PlacementStrategy, placementStrategyFlag, "")
+	cliContext := cli.NewContext(nil, flagSet, nil)
+
+	createServiceTest(
+		t,
+		cliContext,
+		&config.CommandConfig{},
+		&utils.ECSParams{},
+		func(deploymentConfig *ecs.DeploymentConfiguration) {
+			assert.Nil(t, deploymentConfig.MaximumPercent, "DeploymentConfig.MaximumPercent should be nil")
+			assert.Nil(t, deploymentConfig.MinimumHealthyPercent, "DeploymentConfig.MinimumHealthyPercent should be nil")
+		},
+		func(loadBalancer *ecs.LoadBalancer, role string) {
+			assert.Nil(t, loadBalancer, "LoadBalancer should be nil")
+			assert.Empty(t, role, "Role should be empty")
+		},
+		func(launchType string) {
+			assert.NotEqual(t, "FARGATE", launchType)
+		},
+		func(networkConfig *ecs.NetworkConfiguration) {
+			assert.Nil(t, networkConfig, "NetworkConfiguration should be nil")
+		},
+		func(strategy []*ecs.PlacementStrategy) {
+			assert.Equal(t, 1, len(strategy), "PlacementStrategy should have size 1")
+			assert.Equal(t, ecs.PlacementStrategyTypeRandom, *strategy[0].Type, "PlacementStrategy should be random")
+		},
+	)
+}
+
+func TestCreateWithMultiplePlacementStrategies(t *testing.T) {
+	psString := "spread/attribute:ecs.availability-zone,binpack/memory"
```

Comment:
After discussion with the team, we might want to re-jigger the format of this flag value. I don't believe we have a precedent in either the ECS-CLI or the [AWS CLI](https://docs.aws.amazon.com/cli/latest/reference/ecs/run-task.html) for passing in a list of objects in this format. While it is very readable, this format is also a bit limiting if in the future, placement strategies take on a new field.